.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-  "python.pythonPath": "back-end/venv/bin/python3"
+  "python.pythonPath": "C:\\Python32\\python.exe"
```

Comment:
This is for your own computer right? Should probably change this back to what it was.

Reply:
Yes I think so, I'll change it